Tests/LibTTF/CMakeLists.txt

```diff
@@ -3,5 +3,5 @@ set(TEST_SOURCES
 )
 
 foreach(source IN LISTS TEST_SOURCES)
-    serenity_test("${source}" LibGfx)
+    serenity_test("${source}" LibTTF LIBS LibGfx)
```

Comment:
(first commit) did you mean to say "it's tidier" in the commit message?

Reply:
Yes I did!